src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -13,8 +15,32 @@ const checkForErrors = (response: any) => {
     return response;
 };
 
-const sendOphanEvent = (): void => {
-    // TODO stub
+type HasBeenSeen = [boolean, (el: HTMLDivElement) => void];
+
+type OphanAction = 'INSERT' | 'VIEW';
+
+type TestMeta = {
```

Comment:
could be shared with slotBodyEnd too although would require `campaignId` to be optional